Symfony/CS/Fixer/PSR2/ClassDefinitionFixer.php

```diff
@@ -24,17 +25,61 @@
 final class ClassDefinitionFixer extends AbstractFixer
 {
     /**
+     * @var array<string, bool>
+     */
+    private static $defaultConfig = array(
+        // put class declaration on one line
+        'singleLine' => false,
+        // if a classy extends or implements only one element than put it on the same line
+        'singleItemSingleLine' => false,
+        // if an interface extends multiple interfaces declared over multiple lines put each interface on its own line
+        'multiLineExtendsEachSingleLine' => false,
+    );
```

Comment:
depends on what you want, I struggled with the naming of the options so I'm open for other names :)